admin/tests/integration/components/users/user-detail-personal-information/user-overview_test.js

```diff
@@ -3,282 +3,331 @@ import { setupRenderingTest } from 'ember-qunit';
 import { hbs } from 'ember-cli-htmlbars';
 import EmberObject from '@ember/object';
 import { clickByName, render } from '@1024pix/ember-testing-library';
+import Service from '@ember/service';
 
 module('Integration | Component | users | user-detail-personal-information/user-overview', function (hooks) {
   setupRenderingTest(hooks);
 
-  module('When the administrator click on user details', function () {
-    module('update button', function () {
-      test('should display the update button', async function (assert) {
-        // given
-        this.set('user', {
-          firstName: 'John',
-          lastName: 'Harry',
-          email: 'john.harry@example.net',
-          username: 'john.harry0102',
+  module('When the administrator has access to users actions scope', function () {
```

Comment:
Suggestion : rester sur les termes qu'on utilise et ne pas en introduire de nouveau qui amènent une ambiguité.
On avait historiquement des users, on a introduit des admin membres, quand je  "the administrator" j'ai des doutes sur ce que c'est. Un super admin ? Ou juste un admin member ?